File: projects/scriptorium/my-app/pages/api/executeDocker/Dockerfile.c
# Use official GCC image
FROM gcc:latest

# Set the working directory inside the container
WORKDIR /usr/src/app

# Copy the C code into the container as 'temp.c'
COPY temp.c .

# Command to compile and run the C code
CMD ["sh", "-c", "gcc temp.c -o temp && ./temp"]

File: projects/scriptorium/my-app/pages/api/executeDocker/Dockerfile.go
# Use official Go image
FROM golang:1.20-alpine

# Set the working directory inside the container
WORKDIR /usr/src/app

# Copy the Go code into the container as 'temp.go'
COPY temp.go .

# Command to build and run the Go code
CMD ["go", "run", "temp.go"]
